modules/ossm-migrating-hub-recommendations.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/ossm-migrating-from-2-to-3-assembly.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-migrating-hub-recommendations_{context}"]
+= Recommendations
+
+Given the nature of migrating to a brand new operator, the following recommendations are to help limit the risk of misconfigurations or possible conflicts:
+
+* If you have automated updates configured in the {ocp-product-title} web console, change it to **Manual**. This helps to prevent updates to the {SMProduct} 2 Operator and control planes during your migration.
+
+* Keep the amount of the service mesh configuration changes, such as changes for traffic management or security or adding new workloads or gateways, to a minimum during the migration.
+
+[NOTE]
+====
+If it's necessary to add a new workload namespace during the migration, it must be managed by the {SMProduct} 3 control plane and labeled with `maistra.io/ignore-namespace: "true"` to avoid conflicts between the {SMProduct} 3 control plane and the {SMProduct} 2 `ServiceMeshControlPlane` resource.
```

Comment:
```suggestion
If you need to add a new workload namespace during the migration, it must be managed by the {SMProduct} 3 control plane and labeled with `maistra.io/ignore-namespace: "true"` to avoid conflicts between the {SMProduct} 3 control plane and the {SMProduct} 2 `ServiceMeshControlPlane` resource.
```